pkgdb2/templates/list_actions.html

```diff
@@ -0,0 +1,126 @@
+{% extends "master.html" %}
+
+{% block title %} {{ super() }} {% endblock %}
+
+{%block header %}
+{% if refresh %}
+<meta http-equiv="refresh" content="5" >
+{% endif %}
+{% endblock %}
+
+{%block tag %}admin{% endblock %}
+
+
+{% set options = True %}
+{%block options %}
+<form action="{{ url_for('.admin_actions') }}" method="get">
+    Restrict to package: <input type="text" name="package" /> <br />
+    Restrict to packager: <input type="text" name="packager" /> <br />
+    Restrict to action: <input id="action" type="text" name="action" /> <br />
+    Restrict to Status:
+    <select id="status" name="status">
+        {% for act_status in statuses %}
+        <option {%if act_status == status %}selected{% endif %} value="{{
+            act_status}}">
+            {{ act_status }}
+        </option>
+        {% endfor %}
+    </select>
+    <br />
+    <input type="submit" class="submit positive button" value="filter">
+</form>
+{% endblock %}
+
+{% block content %}
+
+<h1>Actions</h1>
+
+{% if total_page and total_page > 1 %}
+<table>
+    <tr>
+        <td>
+        {% if page > 1%}
+            <a href="{{ url_for('.admin_action') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
```

Comment:
Why don't you use url_for('.admin_action', page=page-1, package=package, ...?
As per the documentation (http://flask.pocoo.org/docs/0.10/api/#flask.url_for), all unknown arguments are added as query parameters.